### PR TITLE
docs: add details to expected plugin configuration values

### PR DESCRIPTION
**Pull request by IppX**
**What this PR does / why we need it**:

While trying to include this plugin in an on prem setup I struggled to understand where I could find the values required in the plugin configuration and faced several API errors simply due to mis-configuration. After browsing the code I finally ended up on the contributing guidelines where it was better explained.

**Which issue(s) this PR fixes**:

This PR should make it more understandable for future users where to find the values expected in the plugin provisioning configuration file. It simply copies over the information from the contribution guideline to the README with minor adjustments.

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ apps:
     name: grafana-synthetic-monitoring-app
     disabled: false
     jsonData:
-      apiHost: https://synthetic-monitoring-api.grafana.net
+      apiHost: https://synthetic-monitoring-api.grafana.net || https://synthetic-monitoring-api-eu-west.grafana.net
```
Comment (electron0zero) [inline README.md:30]:
```suggestion
      apiHost: <synthetic monitoring api url for your grafana cloud stack>
```

This URL is different for each region

File: README.md [hunk 2]
```diff
@@ -39,10 +39,15 @@ apps:
       publisherToken: <metric publisher token from grafana.com>
 ```
 
-Note: you can add a provisioning block per [org](https://grafana.com/docs/grafana/latest/manage-users/server-admin/server-admin-manage-orgs/) to provision the plugin for multiple orgs. You can provide different values for each org block and connect to a different cloud stack per org.
-
-The names of the datasources specified in the provisioning file _must_ match the names of existing datasources in Grafana. If you are also provisioning the datasources using the procedure below, make sure the datasource names match.
+Configuration details:
```
Comment (electron0zero) [inline README.md:42]:
```suggestion
Configuration details:

- `apiHost` expects Synthetic Monitoring API URL, [see more details here](https://github.com/grafana/synthetic-monitoring-api-go-client/blob/main/docs/API.md#api-url)

```

**Review by electron0zero — COMMENTED** [review]
thank you 🙏🏼 

left some sugesstions, rest looks good me

**Comment by rdubrock** [comment]
Thanks a bunch y'all!

**Review by rdubrock — APPROVED** [review]

**Pull request merged**